.jenkins/common_job_properties.groovy

```diff
@@ -20,14 +20,27 @@
 // common properties that are shared among all Jenkins projects.
```

Comment:
This is a continuation of the sentence on the above line -- capitalizing seems wrong here?